doc/api/assert.md

```diff
@@ -741,6 +741,42 @@ let err;
 //     at errorFrame
 ```
 
+## assert.match(string, regexp\[, message\])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `string` {string}
+* `regexp` {RegExp}
+* `message` {string|Error}
+
+> Stability: 1 - Experimental
```

Comment:
Not really convinced this needs to be experimental

Reply:
> Not really convinced this needs to be experimental

It may not need to be experimental, but it does no harm and it allows us to take a few releases before committing to the API particulars (the method's name, the order of the arguments, whether or not to have a corresponding `not` method, etc.).